test/formatting.test.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable prefer-arrow-callback */
+
+import chai from 'chai';
+import { formatFromDiscordToIRC, formatFromIRCToDiscord } from '../lib/formatting';
+
+const expect = chai.expect;
+
+describe('Formatting', function() {
+  it('should parse a discord string into an irc string', function() {
```

Comment:
If you've got time I think the tests would be a lot more useful if split into multiple smaller ones, testing things like with or without an ending reset and similar.

Also, the rest of the tests use `should` - consistency is good :) 
If you want to you can use an arrow function here as well, since you don't use `this` inside the test cases.